framework/scripts/wazuh-clusterd.py

```diff
@@ -3,6 +3,9 @@
 # Copyright (C) 2015-2020, Wazuh Inc.
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+from api import configuration
+configuration.api_conf.update(configuration.read_yaml_config())
+
```

Comment:
Is this necessary? If it is not, make sure to change `api/api/configuration.py` as well.

Reply:
For now it would be necessary. We have removed the loading of the `api.yaml` file from the `api.configuration` module. With this, we avoid reloading the file every time that module is imported. However, it is still necessary to load it before the rest of the modules are imported in these two scripts:
- wazuh-apid.py
- wazuh-clusterd.py